configs/approval.json

```
@@ -1,5 +1,23 @@
 {
   "roles": [
+    {
+      "name": "Approval User",
+      "description": "An approval user role which grants permissions to read/create a request.",
+      "system": true,
+      "platform_default": true,
+      "version": 1,
+      "access": [
+        {
+          "permission": "approval:requests:read"
+        },
+        {
+          "permission": "approval:requests.create"
```

Is this supposed to be `approval:requests:create`?
```suggestion
          "permission": "approval:requests:create"
```